app/experimenter/jetstream/client.py

```diff
@@ -171,4 +180,19 @@ def get_experiment_data(experiment):
 
         experiment_data[window] = transformed_data
 
+    errors_by_metric = {}
+    errors_experiment_overall = []
+    if experiment_errors is not None:
+        for err in experiment_errors:
+            if "metric" in err and err.get("metric") is not None:
+                if err.get("metric") not in errors_by_metric:
+                    errors_by_metric[err.get("metric")] = []
+                errors_by_metric[err.get("metric")].append(err)
```

Comment:
Doing the same look up 4 times here, could've pulled it out into

```py
err_metric = err.get('metric')
```

Not critical just a suggestion.